.kokoro/continuous.cfg

```diff
@@ -0,0 +1,24 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```

Comment:
Does this need an Apache header?

Reply:
I don't think so, Kokoro .cfg files in other repo's don't have them. ([google-cloud-python](https://github.com/googleapis/google-cloud-python/tree/master/.kokoro/continuous), [google-cloud-java](https://github.com/googleapis/google-cloud-java/tree/master/.kokoro))